docs/rest_api.md

```diff
@@ -415,10 +415,15 @@ see the documentation of the OBM driver in question (read `docs/obm-drivers.md`)
 `PUT /node/<node>`
 
 Request Body:
-	{"obm": { "type": <obm-subtype>,
-		<additional sub-type specific values>}
-	 "metadata": {"label_1": "value_1",
-	 	     "label_2": "value_2"} (Optional)
+
+	{
+        "obm": {
+             "type": <obm-subtype>, <additional sub-type specific values>
+            }
+	    "metadata": {
+            "label_1": "value_1", 
+            "label_2": "value_2"
+            } (Optional)
```

Comment:
I'm not sure I think this indentation is any better.

Reply:
I followed the suggested format in the very beginning. There is an example in there.

Reply:
It's probably easier for me to just submit a pr with the diff I have in mind than try to explain it in english; I'll do that once this is merged.

Reply:
Edit: what's going on here is we're mixing tabs & spaces; if you render tabs as 4 spaces it looks right. I'm going to submit a PR getting rid of the tabs.

Reply:
Ok. Sorry if this became an issue. Actually, I just parsed / re-indented this JSON-like format thought Visual Studio.

Reply:
`python -m json.tool` formats things in the correct style. A formatting tool might not necessarily work here though, since this isn't quite valid json.